### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by bilalaslamseattle** [Netflix/metacat#245]
Project needs documentation / wiki. Even a PDF would help :)

**Comment by ajoymajumdar** [comment]
Yes, I agree. At present, we just have the README. Since we are busy with other efforts, we are not able to spend much time on the documentation. Do you have any question in particular?

**Comment by bilalaslamseattle** [comment]
@ajoymajumdar I don't have a specific question. However, my feedback is that many potential users (certainly me!) will shy away from open source projects that look good but don't have excellent documentation. It makes me think this project was built for a very specific need at Netflix and may not work for the community at large.

**Comment by geota** [comment]
@bilalaslamseattle  we have appetite to explore using Metacat as a replacement for our in-house Metadata service. However, the docs were lacking and we ran into issues getting the docker compose stack to standup a working service. Can you share any of Netflix's plans as it relates to Metatcat? Is the intention to formally open-source it?

I listened to this talk:
https://www.slideshare.net/sawjd/the-netflix-data-platform-now-and-in-the-future-by-kurt-brown

And at the time, it was mentioned that you guys are working on finishing adding 'canonical data types' and then were planning to move forward with open sourcing the project. Any insight on the roadmap for this would be helpful!

**Comment by ajoymajumdar** [comment]
We never officially open sourced Metacat. We shared our code in Github because a few engineers from Stichfix were interested in the project too. Hence, at present, we do not have any documentation about Metacat. 
Metacat as a product is active at Netflix. We did implement features like notifications and canonical types mentioned by Kurt. The plans to open-source always gets deferred because of our internal commitments. We should be publishing a blog on our Netflix tech site soon. My apologies for not being able to add more details.
Docker issue: Our builds uses the docker steps to run our integrations tests. Let us know what is the issue you are experiencing with the docker compose.

**Comment by ajoymajumdar** [comment]
Just published a blog post about Metacat - [](https://medium.com/netflix-techblog/metacat-making-big-data-discoverable-and-meaningful-at-netflix-56fb36a53520). This should provide insights on the usage of Metacat at Netflix.

**Comment by lynchlee** [comment]
Cloud I spend some time for a docs about this question ?  But where is our home for this Great Project, Metacat ....

**Comment by maixiaohai** [comment]
Hi, I'm an engineer from Xiaomi, We want to explore metacat, but could't find some start-up docs. Could we have more docs?

**Comment by ronakkerman** [comment]
@ajoymajumdar. Hi, we also have an appetite to use Metacat. We read your blog post and comment on documentation, we also installed metacat and it seems that this version does not save the metadata to a database/datastore, it looks like it is caching metadata information of connected databases in memory. Can you confirm this and are there plans to implement a internal permanent database/datastore for this? Thanks!

**Comment by zuokeng** [comment]
Hello, how should I deploy to the server after I package the 1.2 version? Please provide a little help.

**Comment by zuokeng** [comment]
> Hi, I'm an engineer from Xiaomi, We want to explore metacat, but could't find some start-up docs. Could we have more docs?

Hello, I saw on the Internet that your company has carried out the secondary development of metacat and put it into production. If you can help me a little.

**Comment by leiyongfeng-dot** [comment]
Hi, I'm an engineer from bwton, We want to explore metacat, but could't find some start-up docs. Could we have more docs?
